docs/prerequisites/cmake.rst

```diff
@@ -8,4 +8,30 @@ CMake is an open-source family of tools designed to build, test and
 package software.
 
 -  Please download and install the latest binary distribution of CMake
-   for your platform from: https://cmake.org/download/
+   for your platform from `here. <https://cmake.org/download/>`__
+
+
+.. figure:: ../images/cmake/cmake_release.PNG
```

Comment:
Is having an image of the release page really a viable approach? 

I don't think so, because releases happen every now and then and we would need to change it everytime possibly.

Reply:
Just a link to the releases page would be enough I think.

Reply:
Shall I remove the release image?

Reply:
Yes. The screenshot may be misleading as it shows a 3.21.2 as the current release.